Ny oppgave om dictionaries

okpedersen opened the pull request:
Har her brukt 'ordliste' som den norske oversettelsen av 'dictionary', kunne brukt f.eks. 'ordbok' eller den bare 'dictionary'.. Noen tanker?

okpedersen committed:
Ny oppgave om dictionaries

tjesi commented:
Tror kanskje "ordbok" er den beste måten å oversette på. Her
https://docs.google.com/spreadsheets/d/1paxKE2tdCmrizLtdhuQY78qAjiZIVQ8qTHmEoC8TVFc/edit#gid=0
er et dokument med ord og oversettelser som kan være til nytte.
"dictionary" står ikke der enda.

Tjerand

2015-07-09 16:19 GMT+02:00 Ole Kristian Pedersen notifications@github.com:

> Har her brukt 'ordliste' som den norske oversettelsen av 'dictionary',
> 
> ## kunne brukt f.eks. 'ordbok' eller den bare 'dictionary'.. Noen tanker?
> 
> You can view, comment on, or merge this pull request online at:
> 
>   https://github.com/arve0/oppgaver/pull/35
> Commit Summary
> - Ny oppgave om dictionaries
> 
> File Changes
> - _A_ src/python2/ordlister/ordlister.md
>   https://github.com/arve0/oppgaver/pull/35/files#diff-0 (128)
> 
> Patch Links:
> - https://github.com/arve0/oppgaver/pull/35.patch
> - https://github.com/arve0/oppgaver/pull/35.diff
> 
> —
> Reply to this email directly or view it on GitHub
> https://github.com/arve0/oppgaver/pull/35.

okpedersen committed:
Ordliste -> ordbok

arve0 committed:
revidert ordbøker

- noen få skrivefeil
- omformuleringer (tilsvarende #4)
- fjernet `d = {...}` fra suksessive eksempler
- løkke uten eksplisitt `.keys()`
- løkke med `.items()`

okpedersen committed:
Merge pull request #5 from arve0/vg_python_ordlister

revidert ordbøker
